src/deepqmc/conf/task/train.yaml

```diff
@@ -31,15 +31,6 @@ loss_function_factory:
 mols:
   _target_: deepqmc.app.read_molecules
   directory: null
-observable_monitors:
-  - _target_: deepqmc.observable.EnergyMonitor
-    _partial_: false
-    save_samples: true
-    period: 1
-  - _target_: deepqmc.observable.WaveFunctionMonitor
-    _partial_: false
-    save_samples: true
-    period: 1
```

Comment:
> keys_whitelist:
>     - local_energy
This field should now also be obsolete in the config files.  Do you agree that we can remove it?

Reply:
I was actually thinking about removing it. I think it was obsolete even before this change, so I didn't want to touch it since it might have some purpose I'm not aware of. But let's remove it then.

Reply:
Removed `local_energy' from all config files in https://github.com/deepqmc/deepqmc/pull/216/commits/02e1e75bee8de86adc1db681e2dc3e8690894483